code/modules/food/food/snacks.dm

```diff
@@ -1648,7 +1656,7 @@
 /obj/item/weapon/reagent_containers/food/snacks/slimesoup
 	name = "slime soup"
 	desc = "If no water is available, you may substitute tears."
-	icon_state = "rorosoup" //nonexistant? - 3/1/2020 FIXED. roro's live on.
+	icon_state = "slimesoup" //nonexistant? - 3/1/2020 FIXED. roro's live on. - 7/14/2020 - The fuck are you smoking, roro's is stupid, name it slimesoup so it's clear wtf it is.
```

Comment:
I don't think you understand the deranged search I had to find out how long slime soup had been broken in the code. The answer, after literally hours of git blame crap, was since JANUARY 2013, so leaving the final marker of roros in the code was ESSENTIAL

To quote:

> PREHISTORY - Metroid, Sprites named as such
> 
> SEP 17 2012 - Roro Added, Sprites Renamed
> [email censored. it was david]
> 
> NOV 29 2012 - Roro Replaced (slimes)
> ]innocent dev]
> 
> JAN 11 2013 - Toast/Burger renamed 'Jelly' - soup UNTOUCHED
> [david. the criminal. the breaker of ros.]

Reply:
I mean, I get traditions and all that, which is why I left your comment there, but I'd rather stuff be intuitive to newbies lol